test/unit/worldwide_organisation_test.rb

```diff
@@ -4,10 +4,10 @@ class WorldwideOrganisationTest < ActiveSupport::TestCase
   context ".for_location" do
     should "instantiates WorldwideOrganisation objects using data from the API" do
       organisations_data = [
-        { title: "organisation-1-title" },
-        { title: "organisation-2-title" },
+        { "title" => "organisation-1-title", "base_path" => "/world/organisations/organisation-1" },
```

Comment:
It's so the keys remain as strings, rather than being changed to symbols.

Reply:
How come that wasn't needed before, can we put a `.with_indifferent_access` in the library so that users can use whatever hash type they wish?

Reply:
Nice idea.  I've changed these to symbols and updated `gds-api-adapters`.

Reply:
Scrap that idea.  `with_indifferent_access` is a Rails thing, so can't be used in the gem (it'll work if the gem is included in a Rails app, but won't otherwise and the tests will fail).